docs/index.rst

```diff
@@ -38,33 +38,31 @@ pip install --pre --upgrade stellarphot
 Overview
 --------
 
-Using ``stellarphot`` starts with defining a bunch of configuration settings. Some of these,
-like details about your observing location and camera, will change infrequently.
-Others, like what size apertures to use for photometry and where in each image those
-apertures should be placed, may change from night to night.
 
-There is a graphical interface for making all of the settings below. They can also be set
-programmatically in Python or by editing a file in your favorite text editor.
 
-Provide your observatory information
--------------------------------------
+Graphical interface for generating settings
+-------------------------------------------
 
-TBD
+To generate settings using a graphical interface, start Jupyter lab. In the launcher will be a section called
+"Stellarphot" with a link to "Generate Settings". Clicking on this link will open a notebook where you can enter settings.
 
-Provide your camera information
--------------------------------
+Command line interface for generating settings
+----------------------------------------------
 
-TBD
+To generate settings using the command line, run the following command:
 
-Provide a source list
----------------------
+```bash
+stellarphot-settings
+```
+
+This will generate a settings file in the directory in which you run the command
```

Comment:
Is this separate from the one you mentioned going in the OS-dependent preferences directory or does it over-ride it?  You discuss this in the settings.rst file, but didn't know if it made sense to distinguish it here or say to check the settings documentation for more details.